addons/web/static/src/webclient/effects/effect_service.js

```diff
@@ -72,6 +74,7 @@ export const effectService = {
                 fadeout: params.fadeout,
                 id: ++effectId,
                 message: convertRainBowMessage(params.message) || env._t("Well Done!"),
+                messageIsHtml: params.messageIsHtml || false,
```

Comment:
Don't think the `|| false` is necessary here.

Reply:
As above, it's just a detail but I prefer defaulting it to `false` than keeping it undefined.